### DISCUSSION TITLE
How to display GIF in Avalonia

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#11949] [Q&A] [answered]
I would like to know if Avalonia.Labs.Lottie is available for use？

`xmlns:se="clr-namespace:Avalonia.Labs.Lottie;assembly=Avalonia.Labs.Lottie"`
`<se:Lottie Path="/addasoundclientbyavalonia/Assets/Emoji/New_yun.gif"/>`

But it seems like I failed,gif didn't show it.

**Comment by timunie** [comment, marked as answer]
lottie is not for gif iirc. There is a gif viewer somewhere 🤔 see also: #2481

**Reply by pyking-wang** [reply]
Avalonia.GIF has not released the Nuget package, how can I use it?

**Reply by timunie** [reply]
clone source and add it as a project reference

**Reply by pyking-wang** [reply]
A childish question😂sorry
